### PR TITLE
Cta/altro reference

### PR DESCRIPTION
**Pull request by coryalini**
Circle is WIP. Zigzag and line works so i figured id put in an intermediate PR as circle doesnt need to be done.

### CODE REVIEW

File: src/.vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (SchwartzCode) [inline src/.vscode/c_cpp_properties.json:1]: Could you add .vscode/ to the .gitignore?

Comment (SchwartzCode) [inline src/.vscode/c_cpp_properties.json]: Could you add .vscode/ to the .gitignore?

File: src/quadrotor.jl [hunk 2]
```diff
@@ -42,6 +45,242 @@ function visualize!(vis, model, tf::Real, X)
...
+    end
+    obj = Objective(costs_all);
+    
+
+    u0 = @SVector fill(0.5*model.mass/m, m)
+    U_hover = [copy(u0) for k = 1:N-1]; # initial hovering control trajectory
+
+
+    conSet = ConstraintList(n,m,N)
+    bnd = BoundConstraint(n,m, u_min=0.0, u_max=12.0)
```
Comment (SchwartzCode) [inline src/quadrotor.jl:146]: note for later: would be nice if eventually we could connect the thrust bounds between all of these and MPC

**Review by SchwartzCode — APPROVED** [review]
Based on the videos you sent me, this looks good! Are you passing in RobotDynamics' quadrotor as the "model" argument to these functions?

File: src/.vscode/c_cpp_properties.json (hunk 1 above)
Comment (coryalini) [reply src/.vscode/c_cpp_properties.json]: Ahh darn it snuck through. Will do

File: src/quadrotor.jl (hunk 2 above)
Comment (coryalini) [reply src/quadrotor.jl:146]: Will look into it for v2 of altros

**Pull request merged**